src/containerapp/HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+0.3.6
+++++++
+* Added parameter --environment to 'az containerapp list'
+* Added 'az containerapp revision label swap' to swap traffic labels
+* BREAKING CHANGE: 'az containerapp revision list' now shows only active revisions by default, added flag --all to show all revisions
+* Fixed bug with 'az containerapp up' where custom domains would be removed when updating existing containerapp
+* Fixed bug with 'az containerapp auth update' when using --unauthenticated-client-action
+* BREAKING CHANGE: 'az containerapp env certificate upload' does not prompt by default when re-uploading an existing certificate. Added --show-prompt to show prompts on re-upload.
```

Comment:
It is recommended to put the two descriptions of breaking change at the top, so that others can notice them more easily